CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - `GaussianBeam` and `AstigmaticGaussianBeam` default `num_freqs` reset to 1 (it was set to 3 in v2.8.0) and a warning is issued for a broadband, angled beam for which `num_freqs` may not be sufficiently large.
 - Set the maximum `num_freqs` to 20 for all broadband sources (we have been warning about the introduction of this hard limit for a while).
+- Warnings are now generated (instead of errors) when invoking `pml()`, `stable_pml()`,
```

Comment:
it's rather when initializing `PML`, `StablePML` and `Absorber` classes, right? Also maybe instead of being specific about `num_layers<7` we can just say "when invoking x, y, z with fewer layers than recommended. (because we have variable layer requirements for each)